### PR TITLE
Add slack channel link to corrupted disk msg

### PR DESCRIPTION
**Pull request by vadmeste**
## Description
Add slack channel link to corrupted disk msg

## Motivation and Context
Better corrupted disk message.

## How Has This Been Tested?
Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: cmd/storage-errors.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import "errors"
 var errUnexpected = errors.New("Unexpected error, please report this issue at https://github.com/minio/minio/issues")
 
 // errCorruptedFormat - corrupted backend format.
-var errCorruptedFormat = errors.New("corrupted backend format")
+var errCorruptedFormat = errors.New("corrupted backend format, please join https://slack.minio.io to get assistance")
```
Comment (harshavardhana) [inline cmd/storage-errors.go]: `corrupted backend format, please join https://slack.minio.io for assistance`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — DISMISSED** [review]

**Comment by harshavardhana** [comment]
Can you fix the build issues @vadmeste ..

**Comment by vadmeste** [comment]
CI build detected this error:
```
=== RUN   TestServerListenAndServeTLS
panic: test timed out after 15m0s
goroutine 2281784 [running]:
panic(0x1395ee0, 0xc427f98e30)
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/runtime/panic.go:500 +0x1ae
testing.startAlarm.func1()
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/testing/testing.go:918 +0x14e
created by time.goFunc
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/time/sleep.go:154 +0x78
```

I locally tested with go test -count=1000 many times but it didn't show any error, did soemone see this before ?

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
@deekoder waiting on your review.

**Review by deekoder — APPROVED** [review]

**Pull request merged**